actor/src/main/scala-3/org/apache/pekko/util/ByteIterator.scala

```diff
@@ -39,11 +39,11 @@ object ByteIterator {
       extends ByteIterator {
     iterator =>
 
-    @inline final def len: Int = until - from
+    final def len: Int = until - from
```

Comment:
as these code lives in scala-3, should it be `inline def`?

Reply:
Well the thing is, I got feedback from the other PR that we should just remove the `@inline` annotation/`inline` keyword as its misleading.

I mean I can add the `inline` keyword there, but then I would also retain the `@inline` annotation which goes against the premise of this PR. The whole premise of this PR is to remove all of the `@inline`/`inline` (aside from the util wrappers) and if we can demonstrate performance improvements with `inline`/`@inline` then this can be done in future PR's.

Reply:
Thanks for the detail explain.